config/apply/ccsm/collapse.js

```diff
@@ -106,9 +108,21 @@ export function collapseIntoOne(cards, useHtml=false) {
       procedures.length > 0 ||
       diagnosticReports.length > 0 ||
       encounters.length > 0 ||
-      immunizations.length > 0
+      immunizations.length > 0 ||
+      episodeOfCares.length > 0
     ) { markdown = markdown + '\n\n' + '## History' + '\n\n'; }
 
+    if (episodeOfCareString.length > 0) {
+      markdown = markdown + '\n\n' +
+        '### EpisodeOfCare' + '\n\n' +
```

Comment:
Should we consider renaming this display to "Pregnancy Episodes"? I think this could me more intuitive to a user than "EpisodeOfCare", especially since we will only be showing Pregnancy information here.